README.md

```diff
@@ -107,13 +119,6 @@ Also, make sure to understand our contribution guidelines, as explained in
 
 Happy hacking!
 
-# Overview
-
-In this video you can get an overview of some of the features and capabilities
-of Portus.
-
-[![preview](https://cloud.githubusercontent.com/assets/22728/9274870/897410de-4299-11e5-9ebf-c6ecc1ae7733.png)](https://www.youtube.com/watch?v=hGqvYVvdf7U)
-
```

Comment:
This is not the confusing bit ... This video is fine, the original user referred to the ascii video on http://port.us.org